### PR TITLE
fix "for loop initial declaration used outside C99 mode" on openbsd

### PR DESCRIPTION
**Pull request by qbit**
This fix makes libuv compile on openbsd-current 

Note: this is not related to issue # 3016 ( now closed ) I opened for joyent/node

### CODE REVIEW

File: src/unix/openbsd.c [hunk 1]
```diff
@@ -245,7 +245,8 @@ uv_err_t uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   size = sizeof(info);
   which[0] = CTL_KERN;
   which[1] = KERN_CPTIME2;
-  for (int i = 0; i < numcpus; i++) {
+  int i;
```
Comment (bnoordhuis) [inline src/unix/openbsd.c:248]: Shouldn't this move to the function prolog?

Comment (qbit) [inline src/unix/openbsd.c:248]: yep! I will re-request.

Comment (bnoordhuis) [inline src/unix/openbsd.c:248]: @qbit: You can redo your commit (or commit on top and squash), then force-push it and post a "updated!" comment.

**Comment by qbit** [comment]
closing for re-pull

**Pull request closed without merging**